build.sbt

```diff
@@ -0,0 +1,65 @@
+import uk.gov.hmrc.DefaultBuildSettings.integrationTestSettings
+import uk.gov.hmrc.SbtArtifactory
+import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin.publishingSettings
+
+val appName = "plastic-packaging-tax-returns-frontend"
+
+PlayKeys.devSettings := Seq("play.server.http.port" -> "8505")
+
+val silencerVersion = "1.7.0"
+
+lazy val microservice = Project(appName, file("."))
+  .enablePlugins(play.sbt.PlayScala, SbtAutoBuildPlugin, SbtGitVersioning, SbtDistributablesPlugin, SbtArtifactory, SbtWeb)
+  .disablePlugins(JUnitXmlReportPlugin)
+  .settings(
+    majorVersion                     := 0,
+    scalaVersion                     := "2.12.11",
+    libraryDependencies              ++= AppDependencies.compile ++ AppDependencies.test,
+    TwirlKeys.templateImports        ++= Seq(
+      "uk.gov.hmrc.hmrcfrontend.views.html.components._",
+      "uk.gov.hmrc.govukfrontend.views.html.components._",
+      "uk.gov.hmrc.govukfrontend.views.html.helpers._"),
+
+    // ***************
+    // Use the silencer plugin to suppress warnings
+    // You may turn it on for `views` too to suppress warnings from unused imports in compiled twirl templates, but this will hide other warnings.
+    scalacOptions += "-P:silencer:pathFilters=routes",
+    libraryDependencies ++= Seq(
+      compilerPlugin("com.github.ghik" % "silencer-plugin" % silencerVersion cross CrossVersion.full),
+      "com.github.ghik" % "silencer-lib" % silencerVersion % Provided cross CrossVersion.full
+    )
+    // ***************
+  )
+  .settings(publishingSettings: _*)
+  .configs(IntegrationTest)
+  .settings(integrationTestSettings(): _*)
+  .settings(resolvers += Resolver.jcenterRepo)
+  .settings(scoverageSettings)
+  .settings(silencerSettings)
+
+lazy val scoverageSettings: Seq[Setting[_]] = Seq(
+  coverageExcludedPackages := List(
+    "<empty>",
+    "Reverse.*",
+    "metrics\\..*",
+    "features\\..*",
+    "test\\..*",
+    ".*(BuildInfo|Routes|Options).*",
+    "logger.*\\(.*\\)"
+  ).mkString(";"),
+  coverageMinimum := 96,
```

Comment:
:thinking: This may be too high to start with. Happy to go with it, if we struggle to go meet that we should reduce it

Reply:
actually I'll leave it for now, when we start coding there we'll adjust that appropriately